android/app/build.gradle

```diff
@@ -22,35 +22,52 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
+
+def keystoreProperties = new Properties()
+def keystorePropertiesFile = rootProject.file('key.properties')
+if (keystorePropertiesFile.exists()) {
+    keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
+}
+
 android {
     compileSdkVersion 28
 
-    sourceSets {
-        main.java.srcDirs += 'src/main/kotlin'
-    }
-
     lintOptions {
         disable 'InvalidPackage'
     }
 
     defaultConfig {
-        applicationId "com.wamf.farmsmart_flutter"
+        applicationId "co.farmsmart.app"
         minSdkVersion 21
         targetSdkVersion 28
         multiDexEnabled true
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    signingConfigs {
+        release {
+//      Commented to build without release keys
```

Comment:
Instead of commenting, is it possible to have a separate one for debug and a separate one for release?

Reply:
It will take a bit of time as its not as straight forward as if it were just android, 
and i dont want to block isaac / jess with this :)

Reply:
Do you mean release build for the ios? 

My worry is that because it's in backlog, we won't be able to test out the android CI improvements to upload a release apk to Google Play. 

If we only consider Android release configuration, would this be easier?

Reply:
As agreed, this configuration is to be delivered this sprint to ensure Archie won't be blocked.